examples/tensorflow-mnist-pixel-threshold/src/train.py

```diff
@@ -327,9 +327,26 @@ def init_train_flow() -> Flow:
             "evaluate_metrics_tensorflow",
             classifier=classifier,
             dataset=testing_ds,
-            upstream_tasks=[history]
+            upstream_tasks=[history],
         )
-        
+        logged_tensorflow_keras_estimator = pyplugs.call_task(
+            f"{_PLUGINS_IMPORT_PATH}.tracking",
+            "mlflow",
+            "log_tensorflow_keras_estimator",
+            estimator=classifier,
+            model_dir="model",
+            upstream_tasks=[history],
+        )
+
+        logged_tensorflow_keras_estimator = pyplugs.call_task(
```

Comment:
I don't think you need to have this call twice, it just has to exist at least once along with the call below to log_metrics